.github/ISSUE_TEMPLATE/01_crash.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report (major)
+about: A major bug which significantly impacts usability, eg. an unexpected quit, crash, freeze.
+labels: bug: crash,
```

Comment:
There might be a risk of a YAML syntax issue due to the colon? Maybe "bug: crash" instead?

Reply:
Yeah, good catch. The GitHub parser may work around this, but yamllint does suggest that quotes are helpful, including for the name field that contains a `#` :)